connectors/opa/opa_connector.go

```diff
@@ -6,24 +6,26 @@ package main
 import (
 	"context"
 	"encoding/json"
-	"errors"
 	"fmt"
 	"log"
 	"net"
 	"os"
 	"strconv"
-	"time"
 
+	opabl "github.com/ibm/the-mesh-for-data/connectors/opa/opaconn-bl"
```

Comment:
Feel free to ignore, but I'm not a fan of the `bl` term, it also doesn't follow https://blog.golang.org/package-names. 

I would name the main go file `main.go` and the package something shorter (`opaconn` or just `lib` or whatever)

Reply:
renamed opa_connector.go to main.go and renamed opaconn-bl package to lib as suggested. Thanks.